src/medals/utils.cljs

```diff
@@ -2,12 +2,29 @@
   (:require [reagent.core :as r]
             [medals.constants :refer [ALLOWED_SUMMATION_VALUES]]))
 
-
 (defn get-total-count [country]
-  (def filtered-count (r/atom []))
-  (for [item (keys country)]
-    (when (contains? ALLOWED_SUMMATION_VALUES item)
-      (println  filtered-count (get country key)))))
+  (let [filtered-count (r/atom [])]
+    (doseq [item (keys country)]
+      (when (contains? ALLOWED_SUMMATION_VALUES item)
+        (reset! filtered-count (conj @filtered-count (get country item)))))
+    (reduce + @filtered-count)))
```

Comment:
Warning bells should go off in your head if you're using an atom to build up a value within a function. It means you're writing in an imperative style rather than a functional (declarative) style which always means worse readability. [Declarative vs imperative](https://dev.to/ruizb/declarative-vs-imperative-4a7l#:~:text=Declarative%20programming%20is%20a%20paradigm,which%20mutate%20the%20program's%20state.)


With my `set` comment above for ALLOWED_SUMMATION_VALUES you can rewrite this in a much cleaner way. 
```suggestion
  (reduce (fn [total summation-key]
            (+ total (get country summation-key))) 
           0 ALLOWED_SUMMATION_VALUES)
```
or with [Anonymous function shorthand](https://stackoverflow.com/questions/13204993/anonymous-function-shorthand)
```clojure
  (reduce #(+ %1 (get country %2)) 0 ALLOWED_SUMMATION_VALUES)
```
or because [keywords can be used as functions](https://blog.jdriven.com/2020/07/clojure-goodness-keyword-as-function/) you can drop the `get`
```clojure
  (reduce #(+ %1 (%2 country)) 0 ALLOWED_SUMMATION_VALUES)
```

Reply:
Acknowledged